README.md

```diff
@@ -112,6 +112,8 @@ Flags:
                                  The alert severity if none is provided via labels.
       --snmp.version=V2c         SNMP version. V2c and V3 are currently supported.
       --snmp.timeout=5s          SNMP timeout
+      --snmp.custom-base-oid-sub-object=BASE_OID_SUB_OBJECT
```

Comment:
Can you rename to `--snmp.sub-object-oid`, and specify it defaults to `--snmp.trap-default-oid` or `--snmp.trap-oid-label`?

Reply:
Fixed